src/lib/components/Link.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react'
+
+export class Link extends React.PureComponent<React.HTMLProps<{}>> {
+  handleClick: React.MouseEventHandler<{}> = (e) => {
+    if (this.props.onClick) {
+      this.props.onClick(e)
+
+      if (e.defaultPrevented) {
+        return
+      }
+    }
+
+    if (!isModifiedClick(e)) {
+      return
+    }
+
+    // e.preventDefault()
```

Comment:
Is this comment needed still?

Reply:
Placeholder for later. Have documented in new commit.